kotlin-reference-server/src/main/kotlin/org/stellar/reference/event/EventService.kt

```diff
@@ -27,7 +27,7 @@ class EventService {
     if (txnId != null) {
       // filter events with txnId
       return receivedEvents.filter {
-        it.type != AnchorEvent.Type.QUOTE_CREATED.type && it.payload.transaction.id == txnId
+        "quote_created" != it.type && it.payload.transaction.id == txnId
```

Comment:
Hey @MazurakIhor, I think the unresolved reference error you got is better resolved by importing `AnchorEvent`, rather than hard-coding the event type here.

Reply:
You're right. It's weird that https://github.com/stellar/java-stellar-anchor-sdk/pull/1102 didn't catch this issue.
Thank you for fixing this issue!